### PR TITLE
Kubemark hollow-node to support multiple tenant partitions

### PR DESCRIPTION
**Pull request by yb01**
support two tenant partitions in kubemark hollow-node type. currently hardcoded with two tenant partition in scripts.

In Testing...

-->
```release-note

```

### CODE REVIEW

File: cmd/kubemark/hollow-node.go [hunk 1]
```diff
@@ -173,23 +179,29 @@ func run(config *hollowNodeConfig) {
...
 		client[i] = clientFromConfig
 	}
 
 	if config.Morph == "kubelet" {
 		// Start APIServerConfigManager
-		// TODO: Arktos-scale-out: evaluate this function and see if this needs to be done on all tenant partitions
-		//
-		go datapartition.StartAPIServerConfigManagerAndInformerFactory(client[0], wait.NeverStop)
+		for _, clt := range client {
+			go datapartition.StartAPIServerConfigManagerAndInformerFactory(clt, wait.NeverStop)
```
Comment (pdgetrf) [inline cmd/kubemark/hollow-node.go:203]: man this is a long name

Comment (yb01) [reply cmd/kubemark/hollow-node.go:203]: yes. some style. please not this is existing code so i won't intent to change it at all :-)

**Pull request merged**